frontend/templates/apply/apply-base.html.jinja

```diff
@@ -0,0 +1,86 @@
+{% extends "fs-base.html.jinja" %}
+
+
+{% from "govuk_frontend_jinja/components/back-link/macro.html" import govukBackLink %}
+{% from "govuk_frontend_jinja/components/cookie-banner/macro.html" import govukCookieBanner %}
+{% from "govuk_frontend_jinja/components/skip-link/macro.html" import govukSkipLink %}
+{% from "govuk_frontend_jinja/components/header/macro.html" import govukHeader %}
+{% from "govuk_frontend_jinja/components/phase-banner/macro.html" import govukPhaseBanner %}
+{% from "govuk_frontend_jinja/components/footer/macro.html" import govukFooter %}
+
+{# TODO sort out service name, use existing mechanism for now #}
+{% set serviceName = gettext("Access Funding") if is_error else service_title or get_service_title() %}
```

Comment:
I might say that this is a thing we shouldn't carry over into the new world - can we make a simpler choice here from the start? This could be one of three things and makes it hard to track down where the service title is coming from.

If we don't have enough info to make that decision right now, that's maybe ok - but it feels like a bit of complexity we should avoid carrying over if at all possible. We'll probably need to do lots of little tidy-ups like this as we go.

Reply:
I don't think we've had a decision yet from content, but I've chased, so I was keeping it as-is for now

Reply:
Just using `get_service_title()` so we can delay the decision but get rid of the ugly stuff